src/main/java/dk/bec/bookanything/service/FacilityService.java

```diff
@@ -0,0 +1,73 @@
+package dk.bec.bookanything.service;
+
+import dk.bec.bookanything.dto.DayOpenReadDto;
+import dk.bec.bookanything.dto.FeatureReadDto;
+import dk.bec.bookanything.mapper.DayOpenMapper;
+import dk.bec.bookanything.mapper.FeatureMapper;
+import dk.bec.bookanything.model.AddressEntity;
+import dk.bec.bookanything.model.FacilityEntity;
+import dk.bec.bookanything.model.FacilityTypeEntity;
+import dk.bec.bookanything.repository.FacilityRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class FacilityService {
+
+    private final FacilityRepository facilityRepository;
+    private final FeatureMapper featureMapper;
+    private final DayOpenMapper dayOpenMapper;
+
+    public FacilityService(FacilityRepository facilityRepository, FeatureMapper featureMapper, DayOpenMapper dayOpenMapper) {
+        this.facilityRepository = facilityRepository;
+        this.featureMapper = featureMapper;
+        this.dayOpenMapper = dayOpenMapper;
+    }
+
+    public Optional<FacilityEntity> getFacilityById(Long id)
+    {
```

Comment:
lets stick to one way of indeting brackets - so lets agree to have indentation like 

nameOfMerthod(){
....
}

rather than 

nameOfMethod()
{
......
}